README.md

```diff
@@ -1,717 +1,206 @@
-
-### audit.kubernetes.io/pod.allow-read-only-root-filesystem-false
-
-Allows setting `readOnlyRootFilesystem` to `false` to all containers in a pod.
-
-```sh
-kubeaudit.allow.readOnlyRootFilesystemFalse: "WritePermissionsNeeded"
-
-WARN[0000] Allowed setting readOnlyRootFilesystem to false Reason="WritePermissionsNeeded"
+kubeaudit all
 ```
 
-<a name="egress_label"/>
+## Commands
 
-### audit.kubernetes.io/\<namespace-name\>.allow-non-default-deny-egress-network-policy
+| Command          | Description                                                  | Documentation                     |
+| :--------------- | :----------------------------------------------------------- | :-------------------------------- |
+| `all`            | Runs all available auditors, or those specified using a kubeaudit config. | [docs](docs/all.md)  |
+| `autofix`        | Automatically fixes security issues.                         | [docs](docs/autofix.md)           |
 
-Allows absense of `default-deny` egress network policy for that specific namespace.
+### Auditors
 
-<a name="ingress_label"/>
+Auditors can also be run individually.
 
-### audit.kubernetes.io/\<namespace-name\>.allow-non-default-deny-ingress-network-policy
+| Command          | Description                                                              | Documentation                     |
+| :--------------- | :----------------------------------------------------------------------- | :-------------------------------- |
+| `apparmor`       | Finds containers running without AppArmor.                               | [docs](docs/auditors/apparmor.md) |
+| `asat`           | Finds pods using an automatically mounted default service account        | [docs](docs/auditors/asat.md) |
+| `capabilities`   | Finds containers that do not drop the recommended capabilities.          | [docs](docs/auditors/capabilities.md) |
+| `hostns`         | Finds containers that have HostPID, HostIPC or HostNetwork enabled.      | [docs](docs/auditors/hostns.md) |
+| `image`          | Finds containers which do not use the desired version of an image (via the tag). | [docs](docs/auditors/image.md) |
+| `limits`         | Finds containers which exceed the specified CPU and memory limits.       | [docs](docs/auditors/limits.md) |
+| `mountds`        | Finds containers that have docker socket mounted.                        | [docs](docs/auditors/mountds.md) |
+| `netpols`        | Finds namespaces that do not have a default-deny network policy.         | [docs](docs/auditors/netpols.md) |
+| `nonroot`        | Finds containers running as root.                                        | [docs](docs/auditors/nonroot.md) |
+| `privesc`        | Finds containers that allow privilege escalation.                        | [docs](docs/auditors/privesc.md) |
+| `privileged`     | Finds containers that have docker socket mounted.                        | [docs](docs/auditors/privileged.md) |
```

Comment:
I am not sure the text here is right, also I think we are missing the one for the docker socket :)

Reply:
Oops!